### PR TITLE
Add govulncheck to backplane-api CI

### PR DESCRIPTION
**Pull request by Dee-6777**
This PR is for the issue [OSD-18227](https://issues.redhat.com/browse/OSD-18227) to add govulncheck in backplane-api CI check.

### CODE REVIEW

File: ci-operator/config/openshift/backplane-api/openshift-backplane-api-main.yaml [hunk 1]
```diff
@@ -19,6 +19,10 @@ tests:
     make generate
   container:
     from: src
+- as: scan
+  commands: make scan
+  container:
+    from: src
```
Comment (supreeth7) [inline ci-operator/config/openshift/backplane-api/openshift-backplane-api-main.yaml:25]:
```suggestion
  container:
    from: src
  optional: true
```

**Review by supreeth7 — CHANGES_REQUESTED** [review]
Make the check optional, please.

**Comment by Dee-6777** [comment]
The api repository doesn't need a vulnerability check now. Closing this PR

**Pull request closed without merging**